packages/shared/styles/global/_breakpoints-classes.scss

```diff
@@ -0,0 +1,12 @@
+@import "../helpers/breakpoints";
+// Visibility classes
+.smartphone-only {
+  @include for-desktop {
+    display: none !important;
```

Comment:
I really oppose the use of `important` here.

Reply:
Can we find a way to remove it?

Reply:
I would love to remove it :) but I'm afraid it will break some components. I checked few of them and without !important e.g. : SfColor, SfNotification and SfContentPages don't work properly. Have you got any idea how to get rid of them?

Reply:
Let leave it for now I guess. We will fix it in 0.11.0